image.go

```diff
@@ -472,11 +471,11 @@ func (i *image) getManifestPlatform(ctx context.Context, manifest ocispec.Manife
 		return ocispec.Platform{}, err
 	}
 
-	var image ocispec.Image
-	if err := json.Unmarshal(p, &image); err != nil {
+	var img ocispec.Image
+	if err := json.Unmarshal(p, &img); err != nil {
```

Comment:
After pushing I realised that this will conflict with https://github.com/containerd/containerd/pull/7376 (that one already needed a rebase, but happy to rebase if that one gets merged first)

Reply:
Needs a rebase now :smile:.

Reply:
Yup! Tianon won the race 🏁

rebased; PTAL